cypress-tests/cypress/e2e/PaymentUtils/Nexixpay.js

```diff
@@ -16,19 +16,99 @@ const successfulThreeDSTestCardDetails = {
   card_cvc: "396",
 };
 
+const multiUseMandateData = {
+  customer_acceptance: {
+    acceptance_type: "offline",
+    accepted_at: "1963-05-03T04:07:52.723Z",
+    online: {
+      ip_address: "125.0.0.1",
+      user_agent: "amet irure esse",
+    },
+  },
+  mandate_type: {
+    multi_use: {
+      amount: 8000,
+      currency: "EUR",
+    },
+  },
+};
+
+const singleUseMandateData = {
+  customer_acceptance: {
+    acceptance_type: "offline",
+    accepted_at: "1963-05-03T04:07:52.723Z",
+    online: {
+      ip_address: "125.0.0.1",
+      user_agent: "amet irure esse",
+    },
+  },
+  mandate_type: {
+    multi_use: {
+      amount: 8000,
+      currency: "EUR",
+    },
+  },
+};
+
+const payment_method_data_no3ds = {
+  card: {
+    last4: "1111",
+    card_type: null,
+    card_network: null,
+    card_issuer: null,
+    card_issuing_country: null,
+    card_isin: "411111",
+    card_extended_bin: null,
+    card_exp_month: "08",
+    card_exp_year: "25",
+    card_holder_name: null,
+    payment_checks: null,
+    authentication_data: null,
+  },
+};
+
 export const connectorDetails = {
   card_pm: {
     PaymentIntent: {
       Request: {
         currency: "EUR",
-        amount: 3545,
+        amount: 6500,
         customer_acceptance: null,
         setup_future_usage: "on_session",
+        billing: {
```

Comment:
Can you declare billing as const and use it everywhere else the billing address are same